.mailmap

```diff
@@ -91,6 +98,7 @@ Eric Koch                    <koch.eric.w@gmail.com> <koch.eric.w@gmail.com>
 E. Madison Bray              <erik.m.bray@gmail.com>
 E. Madison Bray              <erik.m.bray@gmail.com> <embray@stsci.edu>
 E. Madison Bray              <erik.m.bray@gmail.com> <erik.bray@lri.fr>
+E. Rykoff                    <erykoff@stanford.edu>
```

Comment:
Does this need a rebase? I don't see the `\.` thingy that Simon recommended in the other PR.

Reply:
I didn't add the escapes to the mailmap file, just to the generated sphinx docs. Whether we include the escapes here should maybe be discussed separately as the mailmap can be used to make other lists where the \ might not be needed.